### PR TITLE
Crossword test

### PR DESCRIPTION
**Pull request by gidsg**
Add a new low priority test to check we are indexing crosswords. (To catch recent regression where this was failing).

I assert there has to have been a new crossword within 25 hours to give some grace in case the crossword is not published exactly at midnight. Also I exclude the days after Christmas and Boxing Day as we do not always publish a crossword on those days.

Also changed the Low Priority tests to run at 9am rather than midday.
CC @guardian/content-platforms

### CODE REVIEW

File: app/com/gu/contentapi/sanity/CrosswordsIndexingTest.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      val now = DateTime.now
+      val isAfterChristmas = now.getMonthOfYear == 12 && (now.getDayOfMonth == 26 || now.getDayOfMonth == 27)
+      val twentyFiveHoursAgo = now.minusHours(25)
+      assume(!isAfterChristmas, "Cancelling as Crosswords are not always published over the Christmas period")
+
+      val httpRequest = requestHost("search?tag=type/crossword&order-by=newest").get
+      whenReady(httpRequest) { result =>
+        assume(result.status == 200, "Service is down")
+        val json = Json.parse(result.body)
+        val newestItem = (json \ "response" \ "results")(0)
```
Comment (Nick-Smith) [inline app/com/gu/contentapi/sanity/CrosswordsIndexingTest.scala]: This line will throw an IndexOutOfBounds exception if no results are returned, whcih isn't a very informative/helpful message.
A better approach would be to use headOption, which gets the first item and wraps it in an Option so it won't throw an error if the results are empty, it'll just return None:
 val newestItemOpt = (json \ "response" \ "results").headOption

Then you can check if the item is defined, with some appropriate failure message:
newstItemOpt.isDefined should be(true) 

Then you can either get the newstItem using newsetItemOpt.get or to be really clean you can use:
newestItemOpt foreach { newestItem =>
... rest of the test code goes here.
}

**Comment by gidsg** [comment]
Thanks @Nick-Smith I can't use `headOption` as I'm dealing with a `JSValue` rather than a Scala List but I have implemented it instead with `.asOpt[JsValue]` and made the other changes you suggested.

**Comment by gidsg** [comment]
@Nick-Smith does this look OK to merge now?

**Comment by cb372** [comment]
:+1:

**Pull request merged**